website/hw/hw1.rst

```diff
@@ -92,6 +87,9 @@ on the graders’ inspection of your GitHub repository (but we still need
 to include those tasks on Gradescope so we can grade them as part of
 this homework).
 
+To see how this homework assignment will be graded, please see the
+`Homework 1 Rubric <hw1_rubric.html>`__
```

Comment:
I'm not sure if this will be updated, but currently this is an invalid link. I'm guessing that we have yet to update the file format of the rubric.

Reply:
The link won't work if you're reading through the .rst file on GitHub, but will work when the website is built into HTML.